hard/gf_14/bsg_misc/bsg_reduce_gen.py

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/python
+
+import sys;
+
+print '''
+
+module bsg_rp_gf_14_reduce_and_b4 (input [3:0] i, output o);
```

Comment:
comment with justification (or lack of justification) for using high leakage cells